genie-core/src/main/java/com/netflix/genie/core/jobs/JobConstants.java

```diff
@@ -130,6 +130,12 @@
     public static final String GENIE_DONE_FILE_NAME = "genie/genie.done";
 
     /**
+     * Temporary done filename generated by Genie after running a job as not to conflict/overwrite the
+     * one that 'trap' handler might have created.
+     **/
+    public static final String GENIE_TEMPORARY_DONE_FILE_NAME = "genie/temp.genie.done";
```

Comment:
not sure but I thought most temporary files in *nix were desired to be {filename}.tmp or similar rather than prepending? Not a huge deal either way